scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -735,7 +737,7 @@ object FormatWriter {
   }
 
   private val leadingAsteriskSpace =
-    Pattern.compile("(?<=\\n)\\h*(?=[*][^*])")
+    Pattern.compile("\n\\h*\\*([^*])", Pattern.MULTILINE)
```

Comment:
You are correct, no anchors are in use. I can try without the MULTILINE. Thank you for second set of eyes.

I had convinced myself that it was necessary but am probably all twisted up.
An experiment will soon verify.   Simpler if it can be removed.